### ISSUE TITLE
How do you manage your sleep?

### ISSUE THREAD

**Issue by clementgalidie** [HugoGiraudel/ama#38]
Hey!
Long nights? short nights? polyphasic sleep? Something else?

**Comment by HugoGiraudel** [comment]
I am a heavy sleeper. I could easily hit 10+ hours / night. Unfortunately, life is not that easy. :D

For starters, it's actually quite hard to allocate 10 hours a day on sleeping, especially when you plan on having a life after the work day. It would require to get to bed at 10PM or so, and I cannot do that. I can do so many things in 2 hours (between 10PM and 12PM) that I just cannot resolve at falling asleep at 10.

Secondly, my girlfriend has trouble sleeping, and it's getting worse with the years. She usually wake up between 7AM and 9AM, even on week-ends, and thus no matter at what time she got to bed. While she doesn't wake me up, my sleep is obviously not as good when there is a bit of noise in the flat.

On top of that, as soon as one of us is waken up, our 2 cats think it's time to run and play, so it's getting even harder to sleep.

Long story short, I usually go to bed slightly after midnight or so, and wake up between 8 and 8:30, which gives me a regular 8 hour night. Although it's not a one-shot 8 hours sleeping span, as I use to wake up at least once per night.

**Comment by varemenos** [comment]
Mind if I jump in and ask a question related to this?
Do you ever "Snooze" the alarm?

**Comment by HugoGiraudel** [comment]
A lot. I have 2 alarms scheduled every morning, spaced by 5 minutes. They both ring every 3 minutes for 30 minutes. So it might look like this:

- 8:30 — Alarm 1 rings
- 8:33 — Alarm 1 rings
- 8:35 — Alarm 2 rings
- I start waking up
- 8:36 — Alarm 1 rings
- 8:38 — Alarm 2 rings
- 8:39 — Alarm 1 rings
- Cat 1 jumps on the bed
- 8:41 — Alarm 2 rings
- Cat 1 starts purring
- 8:42 — Alarm 1 rings
- 8:44 — Alarm 2 rings
- I realise I can still sleep a lot
- 8:45 — Alarm 1 rings
- Cat 2 jumps on the bed
- 8:47 — Alarm 2 rings
- Cat 2 sit on my chest
- 8:48 — Alarm 1 rings
- Cat 2 starts purring
- 8:50 — Alarm 2 rings
- Cat 1 lay beside me
- 8:51 — Alarm 1 rings
- 8:53 — Alarm 2 rings
- 8:54 — Alarm 1 rings
- 8:56 — Alarm 2 rings
- 8:57 — Alarm 1 rings
- 8:59 — Alarm 2 rings
- 9:00 — Alarm 1 rings
- 9:02 — Alarm 2 rings
- 9:05 — Alarm 2 rings
